samples/bluetooth/broadcast_audio_sink/Kconfig.sysbuild

```diff
@@ -8,6 +8,7 @@ config NET_CORE_BOARD
 	default "nrf5340dk/nrf5340/cpunet" if "$(BOARD)" = "nrf5340dk"
 	default "nrf5340_audio_dk/nrf5340/cpunet" if "$(BOARD)" = "nrf5340_audio_dk"
 	default "nrf5340bsim_nrf5340_cpunet" if "$(BOARD)" = "nrf5340bsim_nrf5340_cpuapp"
+	default "nrf5340bsim/nrf5340/cpunet" if "$(BOARD)$(BOARD_IDENTIFIER)" = "nrf5340bsim/nrf5340/cpuapp"
```

Comment:
there is a conflict here with https://github.com/zephyrproject-rtos/zephyr/pull/70438 if that PR is merged first then this will need to be updated and have `/` added between the items

Reply:
Thanks for the review @nordicjm 
About this conflicting PR, thank you for pointing it out. Whichever goes second will need to update.